masterlist.yaml

```diff
@@ -7889,3 +7889,14 @@ plugins:
         crc: 0x82DAC22D
         util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/34703)'
         itm: 1
+
+  - name: 'HiddenValleyOverhauled.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/70859/' ]
```

Comment:
While this is a valid method to add url's to plugin entries, please write it like this:
```yaml
  - name: 'HiddenValleyOverhauled.esm'
    url:
      - link: 'https://www.nexusmods.com/newvegas/mods/70859/'
        name: 'Hidden Valley Overhauled'
```

Reply:
Done